src/search/components/SearchForm.js

```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from 'react';
+
+const propTypes = {
+  onSearch: PropTypes.func.isRequired,
+};
+
+class SearchForm extends React.Component {
+  constructor() {
+    super();
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(e) {
```

Comment:
Looks like I didn't have babel-eslint setup correctly, so that's not an issue. I'll update this stuff and hopefully the proposal doesn't change.